maxim/models/prompt.py

```
@@ -42,16 +82,45 @@ def from_dict(obj: Dict[str, Any]):
         content = obj["content"]
+
+        # Handle both string content (backward compatibility) and array content (multimodal)
+        if isinstance(content, str):
+            # String content - keep as is for backward compatibility
+            pass
+        elif isinstance(content, list):
+            # Array content - validate and convert to proper CompletionRequestContent objects
+            content = [
+                (
+                    CompletionRequestTextContent(
+                        type=item["type"], text=item.get("text", "")
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)